src/ReactiveDomain/ReactiveDomain.Tests/Logging/with_event_store.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Threading;
+using EventStore.ClientAPI;
+using ReactiveDomain.Domain;
+using ReactiveDomain.EventStore;
+using ReactiveDomain.Tests.Specifications;
+
+namespace ReactiveDomain.Tests.Logging
+{
+    // ReSharper disable once InconsistentNaming
+    public abstract class with_event_store :
+        CommandBusSpecification,
+        IDisposable
+    {
+        protected EventStoreLoader EventStore;
+        protected IEventStoreConnection TcpEndPointConnection;
+        protected IRepository Repo;
+
+        private static readonly IPEndPoint IntegrationTestTcpEndPoint;
+
+        static with_event_store()
+        {
+            BootStrap.Load();
+            IntegrationTestTcpEndPoint = new IPEndPoint(IPAddress.Loopback, 1113);
+        }
+
+        protected override void Given()
+        {
+
+            TcpEndPointConnection = EventStoreConnection.Create(IntegrationTestTcpEndPoint);
+            EventStore = new EventStoreLoader();
+
+            TcpEndPointConnection.ConnectAsync().Wait();
+
+            //TODO: better path!  Tacky because PerkinElmer\Greylock...
```

Comment:
This needs to be fixed before merging into ReactiveDomain/master. Why not start up a new EventStore node with a unique path that's determined at runtime? This would also solve your "delete" problem since you could shut down the ES node and delete the store files in the Dispose method of this class here.

Reply:
It's already there. That's why I did it. See:
        public GetEventStoreRepositoryIntegrationTests()
This is not the path to the ESData (well it is also that) but the path to the "installed" copy of the ES exe.  I believe our installer puts it there.

Reply:
can we use an in memory eventstore here? I can share examples.

Reply:
Please send examples :-)